javascript/javascript1/week1/firstex.js

```diff
@@ -0,0 +1,56 @@
+var yearOfBirth = 1997
+var yearFuture = 2022
+var age = yearFuture - yearOfBirth
+
+console.log("you will be " + age + " years old in 2022.")
+
+
+var dogYearOfBirth = 2021
+var dogYearFuture = 2045
+var dogYear = dogYearFuture - dogYearOfBirth
+
+var shouldShowResultInDogYears =true;
+
+
+if (shouldShowResultInDogYears) {
+    dogYear *= 7 
+    console.log("your dog will be ${dogYear} dog years in ${dogYearFuture}")
+}
+
+else {
+    console.log("your dog will be ${dogYear} in ${dogYearFuture} in human years")
```

Comment:
These are double-string quotes. You should use backticks to make it works and be able to use variables inside the string

![image](https://i.stack.imgur.com/wJvjd.jpg)